src/GraknClient.jl

```diff
@@ -76,29 +83,59 @@ include("core/CoreClient.jl")
 include("core/CoreSession.jl")
 include("core/CoreTransaction.jl")
 
+# part of common section -- place because of general grpc resulthandling
+include("common/exception/gRPC_Result_Handling.jl")
+
 
 export GraknCoreBlockingClient, GraknClientException
+export contains_database
 
 
 # export  Session, Transaction
 
 ####### pretty printing sectoin ##################
-Base.show(io::IO, blocking_stub::GraknCoreBlockingStub) = Base.print(io,blocking_stub)
 
+Base.show(io::IO, blocking_stub::GraknCoreBlockingStub) = Base.print(io,blocking_stub)
 function Base.print(io::IO, blocking_stub::GraknCoreBlockingStub)
-    Base.print(io, "GraknCoreBlockingStub(open: $(!blocking_stub.impl.channel.session.closed))")
+    print(io, "GraknCoreBlockingStub($(blocking_stub.impl.channel))")
     return nothing
 end
 
-
-function copy_to_proto(from_object, to_proto_struct::Type{T}) where {T<: ProtoType}
-    result_proto = to_proto_struct()
-    for fname in fieldnames(typeof(from_object))
-        if !hasproperty(result_proto, Symbol(fname)) && getfield(from_object,Symbol(fname)) !== nothing
-            setproperty!(result_proto,Symbol(fname),getfield(from_object,Symbol(fname)))
+## Printing UUId stored in Vector in a more readable way
+Base.show(io::IO, id::Array{UInt8,1}) = print(io, id)
+Base.print(io::IO, id::Array{UInt8,1}) = print(io, string(bytes2hex(id)))
+
+## Printing each request in a shorter form except they have speicalized
+## printing options.
+Base.show(io::IO, item::T) where {T<:ProtoType} = print(io, item)
```

Comment:
Just have `item::ProtoType`, the where clause is unnecessary.
Is overloading this even necessary at all?  What is the fallback if it isn't?